### PR TITLE
Remove risks warning

### LINKED ISSUE #1369: Remove "Risks" view from the pool and trade tabs
Currently, in both the pool tab and the trade tab we are linking to an outdated 2020 [article](https://blog.oceanprotocol.com/on-staking-on-data-in-ocean-market-3d8e09eb0a13) about the risks of rug pulls (which we're claiming to have solved). 

We need something new to link to here. Either in the docs, somewhere in the market, or elsewhere.

Trade tab:
Risks 1 - 2022-04-28_13-56

Pool tab:
Risks 2 - 2022-04-28_13-56

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Fixes #1369

Changes proposed in this PR:

- Removing risks warning from add liquidity tab
- Removing risks warning from add trade tab

### CODE REVIEW

File: src/components/Asset/AssetActions/Pool/Add/index.tsx [hunk 1]
```diff
@@ -143,26 +143,11 @@ export default function Add({
         {({ isSubmitting, setSubmitting, submitForm, values, isValid }) => (
           <>
             <div className={styles.addInput}>
-              {isWarningAccepted ? (
```
Comment (kremalicious) [inline src/components/Asset/AssetActions/Pool/Add/index.tsx:146]: we do not need the `isWarningAccepted` state variable anymore

File: src/components/Asset/AssetActions/Trade/FormTrade.tsx [hunk 2]
```diff
@@ -185,28 +185,14 @@ export default function FormTrade({
     >
       {({ isSubmitting, setSubmitting, submitForm, values, isValid }) => (
         <>
-          {isWarningAccepted ? (
```
Comment (kremalicious) [inline src/components/Asset/AssetActions/Trade/FormTrade.tsx:188]: same here, we do not need the `isWarningAccepted` state variable anymore

**Comment by EnzoVezzaro** [comment]
In my case the banner never shows up. I'm not sure we're supposed to get rid of it entirely, I though we would just remove the "with the risks" part of it. 

https://user-images.githubusercontent.com/13741335/167818951-2ebc7124-07e7-4d67-8d1b-c8cacd795d3f.mp4

File: src/components/Asset/AssetActions/Trade/FormTrade.tsx (hunk 2 above)
Comment (claudiaHash) [reply src/components/Asset/AssetActions/Trade/FormTrade.tsx:188]: same with trade

**Comment by kremalicious** [comment]
> Also the banner never shows up. I'm not sure we're supposed to get rid of it entirely, I though we would just remove the "with the risks" part of it.

that is intended 😃 We get rid of thee disclaimer altogether

**Review by kremalicious — APPROVED** [review]
now it's a nice removal PR. And so great getting to those screens now with 1 click less

**Review by EnzoVezzaro — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**